lib/spack/spack/package.py

```diff
@@ -1298,7 +1371,7 @@ def prefix(self):
         """Get the prefix into which this package should be installed."""
         return self.spec.prefix
 
-    @property  # type: ignore[misc]
+    @property  # type: ignore
```

Comment:
I removed that because ignore[misc] now causes flake8 to fail: 

```
==> Running flake8 checks
  lib/spack/spack/package.py:1374: [F821] undefined name 'ignore'
  lib/spack/spack/package.py:1374: [F821] undefined name 'misc'
```

Reply:
Hmm, I use mypy + flake8 in my projects and I've never had them conflict like this before...

Reply:
Added it back, and ignored the error from pyflakes.